dev-docs/howtoguides/release_a_new_version.md

```diff
@@ -216,20 +267,34 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
     I am marking the verification done for this SRU.
     ```
 
-    Change the tags on the bug from `verification-needed` to `verification-done` (including the verification tags for each Ubuntu release).
+    * Change the tags on the bug from `verification-needed` to `verification-done` (including the verification tags for each Ubuntu release).
 
-    e. For any other related Launchpad bugs that are fixed in this release, perform the verification steps necessary for those bugs and mark them `verification-done` as needed. This will likely involve following the test steps, but instead of adding the staging PPA, enabling -proposed.
+    * For any other related Launchpad bugs that are fixed in this release, perform the verification steps necessary for those bugs and mark them `verification-done` as needed. This will likely involve following the test steps, but instead of adding the staging PPA, enabling -proposed.
```

Comment:
```suggestion
    * For any other related Launchpad bugs that are fixed in this release, perform the verification steps necessary for those bugs and mark them `verification-done` as needed. This will likely involve following the test steps, but instead of adding the staging PPA, enabling `-proposed`.
```